src/modules/portfolio/components/market-portfolio-card/market-portfolio-card.jsx

```diff
@@ -112,6 +120,22 @@ export default class MarketPortfolioCard extends Component {
       default:
         localButtonText = "View";
     }
+    const pendingCompleteSetsHash = `pending-${market.id}-${numCompleteSets &&
+      numCompleteSets.fullPrecision}`;
+    const pendingCompleteSetsInfo = transactionsStatus[pendingCompleteSetsHash];
+    const status = pendingCompleteSetsInfo && pendingCompleteSetsInfo.status;
+    let completeSetButtonText = "Sell Complete Sets";
+    switch (status) {
```

Comment:
Minor: this switch stmt comes up twice, it should be in it's own selector. Then we can grow it out for all tx statuses.